### PR TITLE
Introduce weekly dev meeting summaries

### PR DESCRIPTION
**Pull request by btcdrak**
renders /en/meetings #18

todos

  - [x] add menu
  - [x] deploy across languages
  - [x] add credits
  - [x] fix markdown formatting

### CODE REVIEW

**Comment by MarcoFalke** [comment]
>  as a non-technical person such as myself

Is the author mentioned on the rendered page?

**Comment by btcdrak** [comment]
@MarcoFalke updated

**Comment by harding** [comment]
Why were the disclaimers removed in bcb5ac6 ?

**Comment by btcdrak** [comment]
It is our duty to make sure that content is factually correct so no disclaimer is necessary, also that text was primarily because of the medium (reddit) as part of being a social post. But I also hadnt finished, I pushed the remaining of my changes which added source and credits to the footer of the document and reformatted the markdown.

**Comment by harding** [comment]
@btcdrak We should, of course, endeavor to only publish accurate information.  However, @G1lius was commendably forthright by telling people he wasn't a Bitcoin developer, that he may have biases, and that the notes may not be complete or accurate for those reasons.  I think it's reasonable for us to keep those disclaimers.

(For reference, the Bitcoin.org developer documentation was written by experienced programmers who had reasonably good understanding of Bitcoin[1], but we put a highly visible disclaimer on it from day one---and it's paid off time and time again, as not a month as gone by when some flaw wasn't found.  We still endeavor for accuracy by fixing those flaws and trying to making sure new documentation is peer reviewed, but Bitcoin turns out to be remarkably difficult to pin down using plain English.)

[1] Or, at least in my case, thought they did.  In hindsight I see my hubris.

**Comment by btcdrak** [comment]
@harding He was being humble and cautious because no-one was proof reading his material. It could be reasonably argued that just about everything should have a disclaimer on it. These are not reddit posts and have been re-purposed for this medium. Each summary has source links to the the original mailing list posts in the credits section, but those, as is, do not fit this format.

**Comment by btcdrak** [comment]
@harding I have extended the credit line to include his original disclaimer.

**Comment by harding** [comment]
@btcdrak thanks!

File: _posts/en/meetings/2015-10-01-meeting.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+When a transaction is relayed across the network it is held by the nodes in memory, until it gets into a block. All these transactions that sit in memory are called the memorypool or mempool for short.
+Like we could see during the spam-attack if there's a big back-log of transactions that couldn't make it in the blockchain this mempool can get pretty big resulting in nodes crashing.
+
+To stop this from happening devs are trying to find a way to limit this mempool, so a mechanism to reject and/or remove transactions from the mempool. The hard part here is to make it so nodes can't be attacked by abusing this mechanism.
+
+There are multiple worked out ideas for this, namely:   
+[Limit mempool by throwing away the cheapest txn and setting min realy fee to it](https://github.com/bitcoin/bitcoin/pull/6722)  
+[Mempool limiting with descendant package tracking](https://github.com/bitcoin/bitcoin/pull/6557)  
+[exponential rising effective min relay feerate](https://github.com/bitcoin/bitcoin/pull/6673)
```
Comment (harding) [inline _posts/en/meetings/2015-10-01-meeting.md]: I get the feeling the above should be a list, or should at least be a comma-separated serialized list.

**Comment by jonasschnelli** [comment]
ACK 1446169f52dc88e9fb8225a789778261e2e116f4 (didn't read every meeting log in detail).

Maybe (in another PR) we should mention that the meeting logs are automatic generated.

**Pull request merged**